### DISCUSSION TITLE
Handling multiple classes

### DISCUSSION THREAD

**Discussion by imohanvadivel** [vercel/next.js#23708] [Help]
Hey, I'm new to NEXTJS, have few doubts reg handling multiple classes.

1. Is there is a way to add both global style and css-module style to an same element

``` 
<div className={style.details, 'card-default'}>
    <p>{name}</p>
</div>
```

2. Is there is a way to add two css-module style to an element
```
<div className={style.details, style.card}>
    <p>{name}</p>
</div>
```

**Comment by sebas2day** [comment]
You could use the `classnames` package for this:
https://github.com/JedWatson/classnames#alternate-bind-version-for-css-modules
